### PR TITLE
Change default propagation to datadog,tracecontext

### PR DESCRIPTION
**Pull request by bm1549**
### What does this PR do? What is the motivation?

Updates the default propagation style across languages

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

**Review by zacharycmontoya — APPROVED** [review]
LGTM from the .NET perspective

**Review by estherk15 — APPROVED** [review]
@bm1549 Thanks for updating this! Let me know if this is ready to merge!

File: content/en/tracing/trace_collection/trace_context_propagation/_index.md [hunk 1]
```diff
@@ -16,19 +16,7 @@ further_reading:
...
-| Java     | `datadog,tracecontext`             |
-| Python   | `tracecontext,datadog`             |
-| Ruby     | `Datadog,b3multi,b3,tracecontext`  |
-| Go       | `tracecontext,datadog`             |
-| Node.js  | `datadog,tracecontext`             |
-| PHP      | `tracecontext,datadog`             |
-| .NET     | `tracecontext,datadog`             |
-| C++      | `datadog,tracecontext`             |
-| Envoy or nginx proxies | `tracecontext,Datadog` |
+W3C Trace Context propagation is available for all supported languages. The default trace propagation style for all languages is `datadog,tracecontext`. Envoy and nginx proxies use the default of `tracecontext,Datadog`.
```
Comment (bouwkast) [inline content/en/tracing/trace_collection/trace_context_propagation/_index.md:19]: I wasn't aware of the `envoy` and `nginx` defaults and can't find them searching, but I know we didn't update those for the default style, so it is likely accurate as is.

**Review by bouwkast — APPROVED** [review]
Looks good to me

Left a comment on the `envoy` and `nginx` mainly wasn't aware of them but I'd imagine it is still accurate

**Review by mcculls — APPROVED** [review]

**Pull request merged**